mirrors.d/mirror.raiolanetworks.com.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: mirror.raiolanetworks.com
+address:
+  http: http://mirror.raiolanetworks.com/almalinux/
```

Comment:
This auto-redirect to https.  http definition should be dropped, or the https redirect should be removed.

Reply:
It was an error in the server configuration. You can now access via http without problems:

```
curl -I http://mirror.raiolanetworks.com/mariadb/
HTTP/1.1 200 OK
Content-Type: text/html; charset=utf-8
Last-Modified: Wed, 10 Jul 2024 14:59:05 GMT
Server: Caddy
Vary: Accept, Accept-Encoding
X-Robots-Tag: noindex
Date: Wed, 10 Jul 2024 14:53:55 GMT
```